src/components/DiffView/index.tsx

```diff
@@ -197,17 +204,29 @@ export class DiffViewBase extends React.Component<Props> {
         messages = selectedMessageMap.byLine[line];
       }
 
-      let widget = null;
+      let widget = enableCommenting && line && !change.isDelete && (
```

Comment:
In the rare (and probably unlikely) case that line is `0` then the fragment below will put a zero in the HTML which is not what you want. I suggest using a ternary and setting it to `null` when false.